tests/phpt/error_handler_refuses_negative_value.phpt

```diff
@@ -0,0 +1,27 @@
+--TEST--
+Test that the error handler is called regardless of the current `error_reporting` setting
```

Comment:
It's not, since the handler is a singleton, so it would be already registered by previous tests. The PHPT is run in isolation so there no such issue.

Reply:
Then I believe that every single test of the error handler that we have in PHPUnit should be moved to PHPT or we should somehow (e.g. with reflection) unset the singleton variable or we will leak state between multiple tests and this is not good. Since we made the handler singleton here it's the correct PR to do it, regardless of the solution we choose